src/utils/rfqm_gas_estimate_utils.ts

```diff
@@ -0,0 +1,16 @@
+import { RFQM_TX_GAS_ESTIMATE } from '../constants';
+
+const TOKEN_GAS_PREMIUM: Record<string, number> = {
+    '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48': 20e3, // USDC
+    '0xdac17f958d2ee523a2206206994597c13d831ec7': 20e3, // USDT
```

Comment:
What about WBTC? Does that have an overhead?

Reply:
not from what I have seen, but if the numbers start to show that it elevates gas costs, we can add an entry